package.json

```diff
@@ -20,7 +20,9 @@
     "e2e:run": "cypress run",
     "e2e:open": "cypress open",
     "docgen": "node docs/scripts/docgen.js",
-    "lint": "eslint './**/*.ts*' './**/*.js' './**/*.jsx'",
+    "lint": "eslint . --cache --report-unused-disable-directives --ext .ts,.tsx,.jsx",
+    "lint:ci": "eslint . --report-unused-disable-directives --ext .ts,.tsx,.jsx",
+    "lint:fix": "eslint . --cache --fix --ext .ts,.tsx,.jsx",
```

Comment:
What the motivation of duplicating commands? If we can simply do 
`yarn lint --report-unused-disable-directive`  or  `yarn lint --fix` thanks to yarn

Reply:
I was going after `--report-unused-disable-directive`, no preference otherwise